modules/ROOT/pages/dataweave-indexed-readers.adoc

```diff
@@ -0,0 +1,51 @@
+= Indexed Readers in DataWeave
+
+In DataWeave some readers of input data (such as XML, JSON, and CSV) support an indexed strategy to avoid loading the whole document in memory while still allowing random access just like in the in-memory strategy. Indexed readers are used when the input is larger than a certain configurable threshold or if the `indexedReader` setting is set to `true`.
```

Comment:
1. In DataWeave some readers > In DataWeave**,** some readers...
2. Indexed readers are used > DataWeave uses indexed readers (confirm if this is true, because it seems that DW automatically uses indexed readers when the input is larger than the limit according to this)